frontend/src/screens/learner.tsx

```diff
@@ -201,45 +199,99 @@ export const DesktopStudyCards: FC<{studies: ParticipantStudy[]}> = ({ studies }
 }
 
 export const StudiesByLearningPath: FC<{filteredStudies: ParticipantStudy[]}> = ({ filteredStudies }) => {
-    const [learningPaths, studiesByLearningPath] = useMemo(() => {
+    const [learningPaths, studiesByLearningPath, completedStudiesByLearningPath] = useMemo(() => {
         return [
             orderBy(
                 (uniqBy(filteredStudies.map(fs => fs.learningPath), (lp) => lp?.label)),
                 ['completed', 'order'],
                 ['asc', 'asc']
             ),
-            groupBy(filteredStudies, (study) => study.learningPath?.label),
+            groupBy(filteredStudies, (study) => {
+                return study.learningPath?.label
+            }),
+            groupBy(filter(filteredStudies, (study) => study.completedAt != null), (study) => {
+                return study.learningPath?.label
+            }),
         ]
     }, [filteredStudies])
 
     const isMobile = useIsMobileDevice()
 
+    type LearningPathRefs = {
+        [key: string]: HTMLElement | null;
+    };
+
+    const learningPathRefs = useRef<LearningPathRefs>({})
+
+    const scrollToLearningPath = (learningPath: string) => {
+        learningPathRefs.current[learningPath]?.scrollIntoView({ behavior: 'smooth', block: 'start' })
+    }
+
+    const [hoveredLearningPath, setHoveredLearningPath] = useState<string | null>(null)
```

Comment:
is this logic/css update required in the designs? i wonder if we could do this with css instead of useRef? in general i try to avoid useref as much as possible cause its one more thing to manage